### PR TITLE
Add subpath handling to FAQ

### LINKED ISSUE #252: Import a sub-path of a niv-managed package?
I'm stuck trying to use https://github.com/Mic92/sops-nix: The readme indicates that to use the `sops` module as a library, you have to add `<sops-nix/modules/sops>` to your imports [here](https://github.com/Mic92/sops-nix#5-deploy); but since I pinned the package to my project with niv, nix-build can't find this module.

I've made a ~minimal repo that, when checked out, fails to build in the way I describe: https://github.com/antifuchs/niv-module-import-question/blob/master/configuration.nix#L3 is the place where I try to import that sops-nix module, but am failing. Feel free to check it out and run `nix-build` in it. It errors with:

```
error: file 'sops-nix/modules/sops' was not found in the Nix search path (add it using $NIX_PATH or -I), at /Users/asf/Mess/current/sops-sobs/configuration.nix:3:15
(use '--show-trace' to show detailed location information)
```

Things I've tried:

* Using `sources.sops-nix`: that gets me the [top-level definition of sops-nix](https://github.com/Mic92/sops-nix/blob/master/default.nix), packages.

* I tried finding out how to change the nix-path to add niv-managed sources, but couldn't find out how to do that. Is that something that's supported?

Anyway, I'm stumped. Something tells me this should work, but none of the examples or bits of documentation I found have made it clear to me what I need to do to make it work.

### PR DESCRIPTION
**Pull request by nmattia**
Closes #252 

@antifuchs @Mic92 how does [this](https://github.com/nmattia/niv/blob/78780df4e3103358e6d6b58bc4b82c0f98047b93/README.md#how-do-i-import-a-subpath-of-a-source) look?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -417,3 +417,16 @@ netrc-file = /PATH/TO/.netrc
 GITHUB_TOKEN=$YOUR_GITHUB_TOKEN niv add ...
 ```
 
+### How do I import a subpath of a source?
```
Comment (Mic92) [inline README.md:420]: Maybe also include how this can be used for NixOS modules, part from that it looks good to me.

Comment (nmattia) [reply README.md:420]: Ah! And how are they used? :stuck_out_tongue:

Comment (Mic92) [reply README.md:420]:
```nix
let 
  sources = import ./nix/sources.nix;
in {
  imports = [ "${sources.package}/path/to/module" ];
}
```

File: README.tpl.md [hunk 2]
```diff
@@ -294,3 +294,16 @@ netrc-file = /PATH/TO/.netrc
...
+In order to use the directory `dir` of a `my-package`, use the following
+pattern:
+
+``` nix
+let
+  sources = import ./nix/sources.nix;
+in sources.my-package + "/dir"
+```
+
+where `sources.my-package` is the root directory, and `+ "/dir"` appends the
```
Comment (antifuchs) [inline README.tpl.md:308]: This is great! For some added clarity, I would suggest the following:

```suggestion
in this example, `sources.my-package` becomes `my-package`'s root directory, and `+ "/dir"` appends the
```

**Comment by antifuchs** [comment]
That's fantastic work, thanks for getting this out so quickly!

File: README.tpl.md (hunk 2 above)
Comment (nmattia) [reply README.tpl.md:308]: That's great! I never know what's clear and what's not.

File: README.md (hunk 1 above)
Comment (nmattia) [reply README.md:420]: Added!

**Pull request merged**